software/java/en.md

```diff
@@ -14,42 +14,42 @@ You can set up Java by following the instructions below:
 
 ## JRE
 
-Grab the Java Runtime Environment (JRE) as `.tar.gz` from the [Oracle Download Page](https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
+Grab the newest Java Runtime Environment (JRE) as `.tar.gz` from the [Oracle Download Page](https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
 
 Extract JRE and move it to /opt:
 
 ``` bash
 cd ~/Downloads
-tar xf jre-8u181-linux-x64.tar.gz
+tar xf jre-8u202-linux-x64.tar.gz
 sudo mkdir -p /opt
-sudo mv jre1.8.0_181 /opt/
-sudo ln -svf /opt/jre1.8.0_181/bin/java /usr/bin/java
+sudo mv jre1.8.0_202 /opt/
+sudo ln -svf /opt/jre1.8.0_202/bin/java /usr/bin/java
```

Comment:
Should just try using wildcards / globbing instead. `sudo ln -svf /opt/jre1.8.0_*/bin/java /usr/bin/java`

Reply:
Done.